src/css/profile/mobile/common/listview.less

```diff
@@ -160,22 +158,27 @@ tau-expandable {
 				color: var(--primary-color);
 			}
 		}
+
+		& > .ui-li-text:first-child {
+			margin-left: 24 * @px_base;
+		}
 		
 		// Divider
 		&.ui-li-divider {
 			&::after {
 				content: "";
 				position: absolute;
-				width: 100%;
+				width: calc(~"100%" - 40 * @px_base);
+				left: 20 * @px_base;
 				bottom: 0;
 				height: 1 * @px_base;
 				background-color: var(--divider-color);
 				opacity: var(--divider-opacity);
 			}
 
 			&.ui-li-has-icon::after {
-				width: calc(~"100%" - 52 * @px_base);
-				left: 52 * @px_base;
+				width: calc(~"100%" - 84 * @px_base);
```

Comment:
just question.
Is responsive applied so -84px is maintained to the specification?

Reply:
Yes. It works correctly because it is calculated base on the 100%.